app/services/answer_sheet_service.py

```diff
@@ -18,29 +18,47 @@ def __init__(self, form_service: FormService, adopter_service: AdopterService):
         self.answer_sheet_collection = self.db.get_database().get_collection("answer_sheets")
         self.logger = logging.getLogger(__name__)
 
-    def create_answer_sheet(self, user_id: str, form_id: str,  answerSheet: AnswerSheetModel, request_id: str = ""):
+    def create_answer_sheet(self, adopter_id: str, form_id: str,  answerSheet: AnswerSheetModel, request_id: str = ""):
         self.logger.info(f"id={request_id} Start service create answer Sheet")
-        
         try:
             with self.db.session.start_transaction():
-                print("######")
-                answerSheet.adopter_id = user_id
+                response = self.db.get_database().get_collection("adopter").find_one({"_id": ObjectId(adopter_id)})
```

Comment:
Adicionar um TODO para melhoria futura, esse código esta acoplado
```suggestion
                # TODO: Melhorar isso com injeção de dependência no futuro
                response = self.db.get_database().get_collection("adopter").find_one({"_id": ObjectId(adopter_id)})
```